tests/platform_tests/conftest.py

```diff
@@ -330,7 +330,8 @@ def advanceboot_loganalyzer(duthosts, rand_one_dut_hostname, request):
     """
     duthost = duthosts[rand_one_dut_hostname]
     test_name = request.node.name
-    if "warm" in test_name:
+    if "warm" in test_name or\
+        ("continuous" in test_name and request.config.getoption("--reboot_type") == "warm"):
```

Comment:
Is `warm` in the test_name of continuous warm reboot test? Or there are any other continuous tests?

Reply:
The name of test is `test_continuous_reboot`. However this change for continuous warm reboot test is not needed in this PR that handles SAD cases only.

I have removed this change in the latest commit, and will raise a new PR to enable log_analyzer in test_continuous_reboot.